.github/workflows/push_network.yml

```diff
@@ -0,0 +1,100 @@
+# push workflow is shared and expected to perform actions after a merge happens
+# on a maintenance branch (default or release). For example updating the
+# draft release-notes.
+# based on great work from
+# https://github.com/T-Systems-MMS/ansible-collection-icinga-director
+name: push_network
+
+on:
+  workflow_call:
+    # https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onworkflow_callinputs
+    inputs:
+      repo:
+        description: Repository url
+        required: true
+        type: string
+    secrets:
+      BOT_PAT:
+        description: Bot secret
+        required: true
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-22.04
+    environment: push
+    steps:
+      - uses: actions/checkout@v3.5.0
+        with:
+          repository: ${{ inputs.repo }} # ansible-collections/ansible.utils
```

Comment:
I didn't, wasn't really sure if it would default to the needed input, so taking it as a variable. 
I can check if this workflow works and optimize the variable in an upcoming PR.